includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -70,10 +70,10 @@ private function getLangWithFallback( $lang ) {
 			'media-kit' => 'http://www.wikia.com/mediakit',
 			'media-kit-contact' => 'http://www.wikia.com/mediakit/contact',
 			'social-facebook' => 'https://www.facebook.com/getfandom',
-			'social-twitter' => 'https://twitter.com/wikia',
-			'social-reddit' => null,
-			'social-youtube' => 'https://www.youtube.com/user/wikia',
-			'social-instagram' => null,
+			'social-twitter' => 'https://twitter.com/getfandom',
+			'social-reddit' => 'https://www.reddit.com/r/wikia',
+			'social-youtube' => 'https://www.youtube.com/channel/UC988qTQImTjO7lUdPfYabgQ',
+			'social-instagram' => 'https://www.instagram.com/getfandom/',
 			'app-store' => 'https://itunes.apple.com/developer/wikia-inc./id422467077',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "app-store" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)